### ISSUE TITLE
Multiple logfiles for conpot

### ISSUE THREAD

**Issue by t3chn0m4g3** [vorband/ewsposter#6]
The next T-Pot release (Industrial Edition) will run multiple instances of conpot. Each instance must have its own conpot log file...

Logstash currently reads them as such ...
```
# Conpot 
  file {
    path => ["/data/conpot/log/*.json"]
    codec => json
    type => "ConPot"
  }
```
Having an array ["file1", "file2", ...] for `ews.cfg` would be great.

**Comment by vorband** [comment]
implemented in a4233f4aaa109ece4e74e20b3b9d449e128d6f40 similar to logstash via wildcard \*
e.g.

[CONPOT]
conpot = true
nodeid = conpot-community-01
**logfile = /data/conpot/log/conpot\-*.json**

Please test and close issue if works as desired.

**Comment by t3chn0m4g3** [comment]
LGTM (spool is empty as well):
![image](https://user-images.githubusercontent.com/4318452/39694587-200c463c-51e8-11e8-89bd-83bca6baf226.png)

Thank you! 😄
